.github/workflows/electron-pr.yml

```diff
@@ -43,7 +43,17 @@ jobs:
           sudo flatpak install org.electronjs.Electron2.BaseApp/x86_64/23.08 -y
       - name: Set up environment
         uses: ./.github/actions/setup
+      - name: Build Electron for Mac
+        if: ${{ startsWith(matrix.os, 'macos') }}
+        run: ./bin/package-electron
+        env:
+          CSC_KEY_PASSWORD: ${{ secrets.CSC_KEY_PASSWORD }}
```

Comment:
iirc we didn't add the notarization here because it caused issues with forks.  Maybe @MatissJanis remembers

Reply:
This is for testing - no reviews yet!!